test/onnx/test_fx_op_consistency.py

```diff
@@ -67,9 +133,280 @@
 #     are now fixed, removed the corresponding xfail.
 #     2b. If a test is not failing consistently, use skip.
 EXPECTED_SKIPS_OR_FAILS: Tuple[onnx_test_common.DecorateMeta, ...] = (
+    skip(
+        "acos", dtypes=onnx_test_common.BOOL_TYPES + onnx_test_common.INT_TYPES,
+        reason=onnx_test_common.reason_onnx_does_not_support("Acos")
+    ),
+    skip(
+        "acosh", dtypes=onnx_test_common.BOOL_TYPES + onnx_test_common.INT_TYPES,
+        reason=onnx_test_common.reason_onnx_does_not_support("Acosh")
+    ),
+    xfail(
+        "add", dtypes=onnx_test_common.BOOL_TYPES,
```

Comment:
This one is simply just type not supported though, with no integral promotion involved.